sdk/nodejs/src/client.ts

```diff
@@ -280,6 +295,43 @@ export class DevCycleClient {
         this.eventQueue.queueEvent(populatedUser, event)
     }
 
+    /**
+     * Call this to obtain a config that is suitable for use in the "bootstrapConfig" option of client-side JS SDKs
+     * Useful for serverside-rendering use cases where the server performs the initial rendering pass, and provides it
+     * to the client along with the DevCycle config to allow hydration
+     * @param user
+     * @param userAgent
+     */
+    async getClientBootstrapConfig(
```

Comment:
Can we add some tests for this?

Reply:
yeah just wanted to verify the approach was sound first, if it looks fine I can write tests